app/main/views/briefs.py

```diff
@@ -67,8 +70,8 @@ def submit_brief_response(brief_id):
         return _render_not_eligible_for_brief_error_page(brief)
 
     if supplier_has_a_brief_response(data_api_client, current_user.supplier_id, brief_id):
-        # TODO redirect to summary of brief response page with flash message
-        abort(404)
+        flash(SUBMIT_SECOND_RESPONSE_ERROR_MESSAGE.format(brief['title']), 'error')
```

Comment:
So now that we have a result page anyway, can we send the supplier there instead of flashing the message? Otherwise there's no way for them to find out what their result was, since they don't remember applying to begin with.